### ISSUE TITLE
BOT not responding to pls search.

### ISSUE THREAD

**Issue by Maverish96** [dankgrinder/dankgrinder#55]
NOTE: BOT here means DankGrinder and not DankMemer.

**Expected behavior**
The BOT was expected to respond with a valid search location.

**Actual behavior**
The BOT does not respond to it and goes into "await resume timed out for: pls search".

**Steps to reproduce**
Just run the bot and wait for the 'pls search' command to be executed. The bot does not respond to it (with search location name).

**Logs**
{"cluster":"default","instance":"Maverish#1141","level":"error","msg":"await resume timed out for: pls search","time":"2021-05-24T15:09:13+05:30"}

**Environment**
Windows 10
Dank Grinder version: 0.11.0

Config:

clusters:
  default:
    master: # mav
      token: "XXX.XXX.XXX"
      channel_id: "SOME NUMBER"

shifts:
  - state: "active"
    duration:
      base: 28800
      variation: 500
  - state: "dormant"
    duration:
      base: 57600
      variation: 500

features:
  commands:
    beg: true
    postmeme: true
    search: true
    highlow: true
    fish: true
    hunt: true
  custom_commands:
    - value: "pls daily"
    - value: "pls use lifesaver"
    - value: "pls use horse"
      interval: 1200 
    - value: "pls use zza"
      interval: 1800
      amount: 0
    - value: "pls dig"
      interval: 0
      amount: 0

  auto_buy:
    fishing_pole: false
    hunting_rifle: false
    laptop: true
  auto_sell:
    enable: false
    items:
      - "boar"
      - "dragon"
      - "duck"
      - "fish"
      - "deer"
      - "rabbit"
      - "skunk"
  auto_gift:
    enable: false
    items:
      - "bank"
      - "pizza"
      - 
  # REMOVED AUTO BJ CODE BECAUSE IT'LL TAKE TOO MUCH SPACE HERE..

  auto_tidepod:
    enable: false
    buy_lifesaver_on_death: false
  balance_check:
    enable: false
    interval: 180
  log_to_file: true
  verbose_log_to_stdout: false
  debug: false

compatibility:
  postmeme:
    - "f"
    - "r"
    - "i"
    - "c"
    - "k"
  allowed_searches:
    - "area51"
    - "purse"
    - "glovebox"
    - "fridge"
    - "crawlspace"
    - "bank"
    - "van"
    - "hospital"
    - "basement"
    - "mels room"
    - "couch"
    - "dog"
    - "attic"
    - "tree"
    - "air"
    - "bus"
    - "coat"
    - "dresser"
    - "grass"
    - "laundromat"
    - "mailbox"
    - "pantry"
    - "pocket"
    - "shoe"
    - "sink"
    - "car"
    - "discord"

  search_cancel:
    - "no"
    - "."
  cooldown:
    dig: 42
    beg: 47
    search: 32
    highlow: 32
    postmeme: 42
    fish: 42
    hunt: 42
    blackjack: 13
    sell: 6
    gift: 26
    share: 13
  await_response_timeout: 5 # tried with a higher response timeout but the BOT just wont response...

suspicion_avoidance:
  typing:
    variation: 250
    speed: 450
  message_delay:
    base: 100
    variation: 400

**Comment by scramble8bit** [comment]
im suffering this to, i just disable it lol

**Comment by Maverish96** [comment]
> im suffering this too, i just disable it lol

but you see, I activate my Horseshoe, and with that, I'm easily able to earn 100k in few minutes using the search command... golden phallic, banknotes, and other rare items are easily obtained from pls search, so it's kinda important for it to work properly...

**Comment by V4NSH4J** [comment]
Yes guys, dank Memer had an update and removed the @username from search response which is why dank grinder has stopped responding

**Comment by scramble8bit** [comment]
> Yes guys, dank Memer had an update and removed the @username from search response which is why dank grinder has stopped responding

rip

**Comment by scramble8bit** [comment]
i thought my config was broken

**Comment by empty8005** [comment]
Hey 

> NOTE: BOT here means DankGrinder and not DankMemer.
> 
> **Expected behavior**
> The BOT was expected to respond with a valid search location.
> 
> **Actual behavior**
> The BOT does not respond to it and goes into "await resume timed out for: pls search".
> 
> **Steps to reproduce**
> Just run the bot and wait for the 'pls search' command to be executed. The bot does not respond to it (with search location name).
> 
> **Logs**
> {"cluster":"default","instance":"Maverish#1141","level":"error","msg":"await resume timed out for: pls search","time":"2021-05-24T15:09:13+05:30"}
> 
> **Environment**
> Windows 10
> Dank Grinder version: 0.11.0
> 
> Config:
> 
> clusters:
> default:
> master: # mav
> token: "XXX.XXX.XXX"
> channel_id: "SOME NUMBER"
> 
> shifts:
> 
> * state: "active"
>   duration:
>   base: 28800
>   variation: 500
> * state: "dormant"
>   duration:
>   base: 57600
>   variation: 500
> 
> features:
> commands:
> beg: true
> postmeme: true
> search: true
> highlow: true
> fish: true
> hunt: true
> custom_commands:
> - value: "pls daily"
> - value: "pls use lifesaver"
> - value: "pls use horse"
> interval: 1200
> - value: "pls use zza"
> interval: 1800
> amount: 0
> - value: "pls dig"
> interval: 0
> amount: 0
> 
> auto_buy:
> fishing_pole: false
> hunting_rifle: false
> laptop: true
> auto_sell:
> enable: false
> items:
> - "boar"
> - "dragon"
> - "duck"
> - "fish"
> - "deer"
> - "rabbit"
> - "skunk"
> auto_gift:
> enable: false
> items:
> - "bank"
> - "pizza"
> -
> 
> # REMOVED AUTO BJ CODE BECAUSE IT'LL TAKE TOO MUCH SPACE HERE..
> auto_tidepod:
> enable: false
> buy_lifesaver_on_death: false
> balance_check:
> enable: false
> interval: 180
> log_to_file: true
> verbose_log_to_stdout: false
> debug: false
> 
> compatibility:
> postmeme:
> - "f"
> - "r"
> - "i"
> - "c"
> - "k"
> allowed_searches:
> - "area51"
> - "purse"
> - "glovebox"
> - "fridge"
> - "crawlspace"
> - "bank"
> - "van"
> - "hospital"
> - "basement"
> - "mels room"
> - "couch"
> - "dog"
> - "attic"
> - "tree"
> - "air"
> - "bus"
> - "coat"
> - "dresser"
> - "grass"
> - "laundromat"
> - "mailbox"
> - "pantry"
> - "pocket"
> - "shoe"
> - "sink"
> - "car"
> - "discord"
> 
> search_cancel:
> - "no"
> - "."
> cooldown:
> dig: 42
> beg: 47
> search: 32
> highlow: 32
> postmeme: 42
> fish: 42
> hunt: 42
> blackjack: 13
> sell: 6
> gift: 26
> share: 13
> await_response_timeout: 5 # tried with a higher response timeout but the BOT just wont response...
> 
> suspicion_avoidance:
> typing:
> variation: 250
> speed: 450
> message_delay:
> base: 100
> variation: 400

Why you removed bj codes 😕 I wanted you use this script for me also

**Comment by kevinyang07** [comment]
Yes this is happening to me too. It has something to do with the search being a reply i believe.

**Comment by jordank4** [comment]
Is there any fix yet?

**Comment by Maverish96** [comment]
> Is there any fix yet?

no fix till now

**Comment by Maverish96** [comment]
> Hey
> 
> > NOTE: BOT here means DankGrinder and not DankMemer.
> > **Expected behavior**
> > The BOT was expected to respond with a valid search location.
> > **Actual behavior**
> > The BOT does not respond to it and goes into "await resume timed out for: pls search".
> > **Steps to reproduce**
> > Just run the bot and wait for the 'pls search' command to be executed. The bot does not respond to it (with search location name).
> > **Logs**
> > {"cluster":"default","instance":"Maverish#1141","level":"error","msg":"await resume timed out for: pls search","time":"2021-05-24T15:09:13+05:30"}
> > **Environment**
> > Windows 10
> > Dank Grinder version: 0.11.0
> > Config:
> > clusters:
> > default:
> > master: # mav
> > token: "XXX.XXX.XXX"
> > channel_id: "SOME NUMBER"
> > shifts:
> > 
> > * state: "active"
> >   duration:
> >   base: 28800
> >   variation: 500
> > * state: "dormant"
> >   duration:
> >   base: 57600
> >   variation: 500
> > 
> > features:
> > commands:
> > beg: true
> > postmeme: true
> > search: true
> > highlow: true
> > fish: true
> > hunt: true
> > custom_commands:
> > 
> > * value: "pls daily"
> > * value: "pls use lifesaver"
> > * value: "pls use horse"
> >   interval: 1200
> > * value: "pls use zza"
> >   interval: 1800
> >   amount: 0
> > * value: "pls dig"
> >   interval: 0
> >   amount: 0
> > 
> > auto_buy:
> > fishing_pole: false
> > hunting_rifle: false
> > laptop: true
> > auto_sell:
> > enable: false
> > items:
> > 
> > * "boar"
> > * "dragon"
> > * "duck"
> > * "fish"
> > * "deer"
> > * "rabbit"
> > * "skunk"
> >   auto_gift:
> >   enable: false
> >   items:
> > * "bank"
> > * "pizza"
> > 
> > 
> > # REMOVED AUTO BJ CODE BECAUSE IT'LL TAKE TOO MUCH SPACE HERE..
> > auto_tidepod:
> > enable: false
> > buy_lifesaver_on_death: false
> > balance_check:
> > enable: false
> > interval: 180
> > log_to_file: true
> > verbose_log_to_stdout: false
> > debug: false
> > compatibility:
> > postmeme:
> > 
> > * "f"
> > * "r"
> > * "i"
> > * "c"
> > * "k"
> >   allowed_searches:
> > * "area51"
> > * "purse"
> > * "glovebox"
> > * "fridge"
> > * "crawlspace"
> > * "bank"
> > * "van"
> > * "hospital"
> > * "basement"
> > * "mels room"
> > * "couch"
> > * "dog"
> > * "attic"
> > * "tree"
> > * "air"
> > * "bus"
> > * "coat"
> > * "dresser"
> > * "grass"
> > * "laundromat"
> > * "mailbox"
> > * "pantry"
> > * "pocket"
> > * "shoe"
> > * "sink"
> > * "car"
> > * "discord"
> > 
> > search_cancel:
> > 
> > * "no"
> > * "."
> >   cooldown:
> >   dig: 42
> >   beg: 47
> >   search: 32
> >   highlow: 32
> >   postmeme: 42
> >   fish: 42
> >   hunt: 42
> >   blackjack: 13
> >   sell: 6
> >   gift: 26
> >   share: 13
> >   await_response_timeout: 5 # tried with a higher response timeout but the BOT just wont response...
> > 
> > suspicion_avoidance:
> > typing:
> > variation: 250
> > speed: 450
> > message_delay:
> > base: 100
> > variation: 400
> 
> Why you removed bj codes 😕 I wanted you use this script for me also



[config - Copy.zip](https://github.com/dankgrinder/dankgrinder/files/6541145/config.-.Copy.zip)
THERE YOU GO, MY FRIEND!

**Comment by Warrior-Aviator** [comment]
Please fix the issue as the pls search command is really an important one. All of us are ready to help you out in any possible way if you want Sir. Thank You.

**Comment by Zomboy225** [comment]
> Yes guys, dank Memer had an update and removed the @username from search response which is why dank grinder has stopped responding

are there older versions that we can download??

**Comment by V4NSH4J** [comment]
> > Yes guys, dank Memer had an update and removed the @username from search response which is why dank grinder has stopped responding
> 
> are there older versions that we can download??

I've got a working version on my fork. It has search and autobuy fixed + dig + work + trivia 
www.github.com/v4nsh4j/dankgrinder

**Comment by Zomboy225** [comment]
> > > Yes guys, dank Memer had an update and removed the @username from search response which is why dank grinder has stopped responding
> > 
> > 
> > are there older versions that we can download??
> 
> I've got a working version on my fork. It has search and autobuy fixed + dig + work + trivia
> [www.github.com/v4nsh4j/dankgrinder](http://www.github.com/v4nsh4j/dankgrinder)

wow thats great ill download it right away
